### PR TITLE
Editing User Guide: Commands

### PR DESCRIPTION
**Pull request by bofeng1999**
This PR adds the Command segment to the User Guide.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -32,89 +32,598 @@ In a conventional application cycle, the large influx of applicants makes it cha
...
+  [[ parameter:parameters ]] supplied by you without ambiguity
+* [Placeholders](#) such as `NAME` and `PHONE` shows you what you should place in each portion of the command
+
+Notice that there is a pair of square brackets `[]` surrounding some [[ parameter:parameters ]] like `note/NOTE`
+in the format. This indicates that the parameter is **optional**. Each of these [[ placeholder:placeholders]] in the
+[[ parameter:parameters ]] have a default value based on the commands. These are documented in the [Commands](#commands)
+section for each command.
+
+```
+🚨NOTE
```
Comment (Merrickneo) [inline docs/UserGuide.md:60]: For standardisation purposes, try to make use of the different info blocks that Frederic used.

I've attached them here for your use:

`Alert Blocks`

<div markdown="span" class="alert alert-info" role="alert">:information_source: <strong>Note:</strong>
Highlight and display information you should pay attention to. </div>

<div markdown="span" class="alert alert-success" role="alert">:bulb: <strong>Tip:</strong>
Highlight tips which you might find useful. </div>

<div markdown="span" class="alert alert-danger" role="alert">:exclamation: <strong>Caution:</strong>
Highlight dangers and things to look out for. </div>


Links:
<a href> Phrases underlined in blue </a> are hyperlinks that will bring you to another part of
this documentation that is relevant to the phrase.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -32,89 +32,598 @@ In a conventional application cycle, the large influx of applicants makes it cha
...
+🚨Note
+* If two or more values of the same parameter are provided, only the last value for that parameter will be taken.
+
+
+**Example:**
+
+**Assumption:**
+
+HMHero does not already contain an applicant with the name "Thomas".
+
```
Comment (Merrickneo) [inline docs/UserGuide.md:184]: Maybe you can elaborate on how Individuals are identified by their name and phone number and also how the name provided is case-sensitive.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -32,89 +32,598 @@ In a conventional application cycle, the large influx of applicants makes it cha
...
+
+**Command Input Box:**
+
+`edit 1 p/97833468 n/Tammy note/Python note/Java`
+
+**Command Output Box:**
+
+`Edited Person: Tammy; Phone: 97833468; Email: thomastan@gmail.com;
+Address: 7 Sims Drive (s)543212; Status: SHORTLISTED; Tags: [Python] [Java]`
```
Comment (Merrickneo) [inline docs/UserGuide.md:499]: Not sure if we want to name `notes` as `tags`
@bofeng1999 @Merrickneo @dunliang0513 @Jxleejiaxin @FredericChow00

**Review by Merrickneo — CHANGES_REQUESTED** [review]
Suggested some minor edits to help with consistency!

File: docs/UserGuide.md (hunk 1 above)
Comment (bofeng1999) [reply docs/UserGuide.md:60]: Noted!

File: docs/UserGuide.md (hunk 2 above)
Comment (bofeng1999) [reply docs/UserGuide.md:184]: good point thank you!

**Review by Merrickneo — APPROVED** [review]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -145,89 +145,608 @@ are already familiar with the application!
...
+
+* `add n/ThomaŚ p/91918153 e/thomas@gmail.com a/6 Sims Drivè (s)543230`
+
+  The restrictions of placeholders are not followed.
+
+* `add`
+
+  There is insufficient information provided; you must minimally provide a name.
+
+Find out more about restrictions in the sections [Flags](#flags), [Placeholders](#placeholders) and [Commands](#commands).
```
Comment (FredericChow00) [inline docs/UserGuide.md:224]: Need to change `placeholder` in glossary to `placeholders` so that this hyperlink works

File: docs/UserGuide.md [hunk 5]
```diff
@@ -145,89 +145,608 @@ are already familiar with the application!
...
+
+* [ ] I know the restrictions of the command
+* [ ] I know what [[ parameter:parameters ]] are supplied to the command
+* [ ] I know the [[ flag:flags ]] for each parameter to be supplied
+* [ ] I know the restrictions of each parameter
+* [ ] I know the effects of not specifying each optional flag.
+
+## Commands
+
+This section shares with you on how to use each [[ command ]] in detail.
```
Comment (FredericChow00) [inline docs/UserGuide.md:257]: not too surre if the `[[ ]]` works for the pop-up in these underlined words... need to check after merging

**Review by FredericChow00 — COMMENTED** [review]
Left some comments but LGTM for the first UG Draft!

We will also likely be changing most outputs to pictures once we finalize our GUI

File: docs/UserGuide.md (hunk 3 above)
Comment (dunliang0513) [reply docs/UserGuide.md:623]: Yea I think so

**Pull request merged**